website/docs/cdktf/cli-reference/commands.html.md

```diff
@@ -306,20 +314,20 @@ Create a new Python project and use a specific version of the `cdktf` package.
 $ cdktf init --template="python" --cdktf-version="0.0.1"
 ```
 
-Create a new Typescript project from an existing Terraform codebase. Please be aware that only Typescript is currently supported and that there are [some known limitations](../packages/@cdktf/hcl2cdk/README.md#known-limitations).
+Create a new Typescript project from an existing Terraform codebase. Currently, only Typescript is supported, and there are [some known limitations](to do link to these when in the docs).
```

Comment:
I think we need some clarification here that it's the `--from-terraform-project` flag that is only supported for Typescript (I had to read this a few times to understand this statement wasn't referring to the `init` command

Reply:
Thank you! I will push up a fix for this